### PR TITLE
Colordiff: Create new package

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

**Comment by matthiasdiener** [comment]
Bump.

File: var/spack/repos/builtin/packages/colordiff/package.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    homepage = "https://www.colordiff.org"
+    url      = "https://www.colordiff.org/colordiff-1.0.18.tar.gz"
+
+    version('1.0.18', '07658f09a44f30a3b5c1cea9c132baed')
+
+    depends_on('perl')
+
+    def install(self, spec, prefix):
+        make("INSTALL_DIR=%s/bin" % prefix, "ETC_DIR=%s/etc" % prefix,
```
Comment (healther) [inline var/spack/repos/builtin/packages/colordiff/package.py]: the %s string formatting is deprecated (to quote the python documentation: "Note The formatting operations described here are obsolete and may go away in future versions of Python. Use the new String Formatting in new code.")

In this case you should be able to simply do a
`"INSTALL_DIR=" + prefix.bin, "ETC_DIR=" + prefix.etc, ...`

File: var/spack/repos/builtin/packages/colordiff/package.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
```
Comment (healther) [inline var/spack/repos/builtin/packages/colordiff/package.py]: not sure if this empty line will give a flake8 error or not, typically it's not there

**Comment by matthiasdiener** [comment]
@healther Thanks, should be fixed now.

File: var/spack/repos/builtin/packages/colordiff/package.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    homepage = "https://www.colordiff.org"
+    url      = "https://www.colordiff.org/colordiff-1.0.18.tar.gz"
+
+    version('1.0.18', '07658f09a44f30a3b5c1cea9c132baed')
+
+    depends_on('perl')
+
+    def install(self, spec, prefix):
+        make("INSTALL_DIR=" + prefix.bin, "ETC_DIR=" + prefix.etc, "MAN_DIR=" + prefix.man, 'install', parallel=False)
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/colordiff/package.py]: This line is too long to pass `flake8` style checks on Travis CI:
```
var/spack/repos/builtin/packages/colordiff/package.py:39: [E501] line too long (118 > 79 characters)
```
Just fyi, you can check where CI failed by clicking *Details* on the right of the Travis CI status down below.

**Pull request merged**